### PR TITLE
Fix Danger doesn't add a comment if commented line is not part of diff

### PR DESCRIPTION
**Pull request by Rouby**
This PR should fix the bug mentioned in #1207

### CODE REVIEW

**Comment by orta** [comment]
This feels pretty good to me (aside from the prettier faff, which I can ignore) thanks!

**Comment by orta** [comment]
Shipped in https://github.com/danger/danger-js/releases/tag/11.0.4

File: source/runner/Executor.ts [hunk 1]
```diff
@@ -399,22 +399,28 @@ export class Executor {
...
-      this.sendInlineReview(git, inlineResultsForReview).catch(_e =>
-        inlineResultsForReview.forEach(inlineResult => inlineResultsIntoResults(inlineResult))
-      ),
+      this.sendInlineReview(git, inlineResultsForReview)
+        .then((_r) => emptyDangerResults)
+        .catch((_e) =>
+          inlineResultsForReview
+            .map((inlineResult) => inlineResultsIntoResults(inlineResult))
+            .reduce(mergeResults, emptyResult)
+        ),
```
Comment (fbartho) [inline source/runner/Executor.ts:419]: Great fix! I had a suspicion the root issue was around here, but the 3 times I traced it, I couldn’t figure it out — indeed it looked like everything was being caught & handled..

**Review by fbartho — COMMENTED** [review]
Great work!! This has been grinding my gears.

**Pull request merged**